tests/test_minhash.py

```diff
@@ -1872,6 +1878,21 @@ def test_merge_scaled():
     for k in mh2.hashes:
         assert k in mh3.hashes
 
+def add_is_symmetric():
```

Comment:
if you look at the files view, here, https://github.com/dib-lab/sourmash/pull/1533/files, you'll see that it says `Added lines #L1882 - L1883 were not covered by tests`. This is because this code is not executed by the testing framework, because the function name doesn't start with `test_` :).